src/docs/asciidoc/cdc-configuration.adoc

```diff
@@ -595,6 +595,87 @@ The Eventuate Tram CDC supports `eventuate-tram` and `eventuate-local`
 
 |===
 
+==== Configuring a cleaner
+
+Cdc can automatically remove old messages from message table and received_message table.
+Example configuration:
```

Comment:
Please describe what this configuration does.
Also, please have properties for configuring `received_message` table too

Reply:
ok